### PR TITLE
Move train_all_steer inside bair_car_data

### PR DESCRIPTION
**Pull request by tpankaj**
On the newly imaged bdd3, I've moved train_all_steer.pkl and val_all_steer.pkl inside bair_car_data directory so that the dataset is self-contained. This PR reflects that change.

### CODE REVIEW

File: Data.py [hunk 1]
```diff
@@ -29,10 +29,12 @@ def __init__(self):
                                        self.hdf5_runs_path)
 
         # Load data indexes for training and validation
+        self.train_all_steer_path = ARGS.data_path + '/train_all_steer'
```
Comment (raunakdoesdev) [inline Data.py]: Doesn't make sense to make these class variables if they are not used anywhere else, I would  make them local variables.

**Review by raunakdoesdev — COMMENTED** [review]
Looks good, but once you fix the class variables I can approve.

File: Data.py (hunk 1 above)
Comment (tpankaj) [reply Data.py]: Will do. I wasn't sure either but I followed the pattern of self.hdf5_runs_path.

**Review by raunakdoesdev — APPROVED** [review]
That's perfect thanks, I can clean up the self.hdf5_runs_path in my next PR

**Pull request merged**